src/components/radio/Radio.props.ts

```diff
@@ -0,0 +1,27 @@
+import { TRadioSizeToken } from './Radio.types';
+
+export type PRadio = {
+    /**
+     * whether the radio button is rendered with a label
+     * @default 'Public'
```

Comment:
there is no default defined in the component and IMO we should not define any default value here